### PR TITLE
8.0 product gtin tests

### PR DESCRIPTION
**Pull request by foutoucour**
Implementations of unittests for product_gtins methods

### CODE REVIEW

**Comment by pedrobaeza** [comment]
A little pep8 issue:

```
./product_gtin/tests/test_product_gtin_functions.py:175:1: W391 blank line at end of file
```

but the rest very good.

File: product_gtin/product_gtin.py [hunk 1]
```diff
@@ -44,28 +59,54 @@ def check_ean8(eancode):
...
     check = ((sum/10 + 1) * 10) - sum
-    if check != int(eancode[-1]):
-        return False
-    return True
+
+    return check == int(upccode[-1])
 
 
 def check_ean13(eancode):
+    """Check if the given ean code answer ean8 requirements
```
Comment (lmignon) [inline product_gtin/product_gtin.py]: ean8? not ean13? :smirk:

**Comment by lmignon** [comment]
@foutoucour Thank you for the contributuon and the TESTS! This will avoid regressions like the one introduced by https://github.com/OCA/product-attribute/commit/076022b335ac09c3487326f88250ee0c57f891db (https://github.com/OCA/product-attribute/issues/56)
:+1: Once the comment from @pedrobaeza will be addressed and Travis back to green

File: product_gtin/product_gtin.py [hunk 2]
```diff
@@ -44,28 +59,54 @@ def check_ean8(eancode):
...
+
+    return check == int(upccode[-1])
 
 
 def check_ean13(eancode):
+    """Check if the given ean code answer ean13 requirements
+    For more details:
+    http://en.wikipedia.org/wiki/International_Article_Number_%28EAN%29
+
+    :param eancode: string, ean-8 code
```
Comment (lmignon) [inline product_gtin/product_gtin.py]: @foutoucour you missed this last one ean-8 :smirk:

**Comment by bwrsandman** [comment]
:+1: Looking good

@foutoucour Could you point out what you see that be improved but not done in this PR so they can be added to an issue todo.

**Comment by gurneyalex** [comment]
Nice work, :+1:. Waiting for @foutoucour to provide the feedback asked for by @bwrsandman to merge.

**Comment by foutoucour** [comment]
@gurneyalex how should I provide the feedback? In the README.rst?

**Comment by bwrsandman** [comment]
@foutoucour an issue in the github project would be an appropriate place, but the readme would be as well. Isn't there a section for that?

**Pull request merged**